pkg/pillar/cmd/zedrouter/zedrouter.go

```diff
@@ -44,6 +44,7 @@ const (
 
 // Set from Makefile
 var Version = "No version specified"
+var dnsServers map[string][]net.IP
```

Comment:
Please put this in zedrouterContext. I don't think we should use global variables for things like this.

Reply:
changed